chat-plugins/info.js

```diff
@@ -636,6 +646,33 @@ var commands = exports.commands = {
 		"The parameter 'mega' can be added to search for Mega Evolutions only, and the parameters 'FE' or 'NFE' can be added to search fully or not-fully evolved Pokemon only.",
 		"The order of the parameters does not matter."],
 
+	randpoke: 'randompokemon',
+	randompokemon: function (target) {
+		var targets = target.split(",");
+		var targetsBuffer = [];
+		var qty;
+		for (var i = 0; i < targets.length; i++) {
+			if (!targets[i]) continue;
+			var num = Number(targets[i]);
+			if (Number.isInteger(num)) {
+				if (qty) return this.sendReply("Only specify the number of Pokemon once.");
+				qty = num;
+				if (1 > qty || qty > 15) return this.sendReply("Number of random Pokemon must be between 1 and 15.");
+				targetsBuffer.push("random" + qty);
+			} else {
+				targetsBuffer.push(targets[i]);
+			}
+		}
+		if (!qty) targetsBuffer.push("random1");
+		//Reassigns the target variable to be called in /dexsearch
+		var argumentsBuffer = arguments;
+		argumentsBuffer["0"] = targetsBuffer.join(",");
```

Comment:
I think making some parameters order-dependent does make sense. E.g. look at command namespaces. The amount of random Pokémon can be thought of bound to the ``/randompokemon`` command. For clarity, it may even not use a comma.
(Man, ``Proxy`` would be cool for this, but it's still so far ahead!)

``/dexsearch`` may well tolerate the ``randomX`` argument anywhere imo. Or better: make it ``random X``. Also, I don't think ``/dexsearch random 6`` is confusing at all? It's just a handy extension.